haiku-misc/neonlights/neonlights-0.0.1.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="Screensaver drawing neon lines"
+DESCRIPTION="NeonLights is a screensaver that draws colorful neon lines. \
+It was based loosely on Jared Tarbell's work. Inspired by Atalax's Substrate."
+HOMEPAGE="https://github.com/pulkomandy/neonlights"
+COPYRIGHT="2017 Adrien Destugues"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="f658b45894fe5d2349be4e06c3a0bce359985e27"
+SOURCE_URI="https://github.com/pulkomandy/neonlights/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="9271f3778218bb63430da00d00b099c5d3e9d83b21d6c132533069b237199663"
+SOURCE_DIR="neonlights-$srcGitRev"
```

Comment:
@dash102, I just noticed that the e-mail you used to create your commit is not associated to your GitHub account.
This means that your contribution will not appear to have been authored by you, **_unless_** you:
1) tell GitHub your e-mail account (see the [question mark](https://github.com/settings/emails) icon near the top of this page.
2) and/**_or_** you configure your local git environment by running:
```
git config --global user.email "dash102@users.noreply.github.com"
```

If you chose the second option, you might need to rebase your commit for your new "user.email" setting to be used.
If you see that your commit still shows the blue question mark you can run this:
```
git commit --amend --author="dash102@users.noreply.github.com"
```
and then push your commit again.